compliance_checker/base.py

```diff
@@ -465,7 +465,7 @@ def attr_check(kvp, ds, priority, ret_val, gname=None, var_name=None):
         else:
             check_val = attr_result[1]
             res_tup = other.validate(name, check_val)
-            msgs = [] if res_tup[1] is None else [res_tup[1]]
+        msgs = [] if res_tup[1] is None else res_tup[1]
```

Comment:
See  https://github.com/ioos/compliance-checker/pull/752/commits/72cd4c95714e3024057fbf0119b8aaa349747f3a#diff-8c4403ce509d5b6852be761a2d9a6ba7L84-L93

The return types of these two separate functions can return `str` or a list with one `str` element.  It might be better to fix things there as well.